tests/integration/kubernetes/gha-run.sh

```diff
@@ -464,7 +464,7 @@ function cleanup() {
 	fi
 
 	# Switch back to the default namespace and delete the tests one
-	delete_test_cluster_namespace
+	delete_test_cluster_namespace || true
```

Comment:
Is it OK to bury an error here with `|| true`? Maybe it prints a warning or something, but you could also catch that error case? (Also, unrelated-ish: Why does the comment above say "switch back to the default namespace"; it's not doing that, is it?)

Reply:
So, we need to fully understand why `delete_test_cluster_namespace` would fail.
@ryansavino, do you know?  Or, IOW, what issue are you trying to fix here?

Reply:
This is in the cleanup, and ensuring that the kata deploy cleanup runs. I was running into this scenario when the tests didn't run (namespace wasn't created) or the tests exited in the very beginning.

Not a huge issue, as kata-deploy will try to cleanup before it runs anyways.

Reply:
The cleanup function:
```
function delete_test_cluster_namespace() {
        set_default_cluster_namespace
        kubectl delete namespace "${TEST_CLUSTER_NAMESPACE}"
}
```

I think we can live with this commit in to avoid retriggering CI and yatta-yatta.

Ideally, we should check whether the namespace exists before trying to delete it.  If possible, **please**,  address this in a follow-up PR.

I'm approving this one as it is.